### ISSUE TITLE
Make the Help relevant to the Actual Code

### ISSUE THREAD

**Issue by Smurf-IV** [Wagnerp/Krypton-NET-4.7#17]
It would be nice if the different dll's had different files, and also remove that dotfusticator rubbish from the current chm as well [Check the global namespac(s) in the references section]!

I can do this if you want ?

**Comment by Wagnerp** [comment]
If you can, that'll be a great help thanks. Been spending hours trying to find a way to update the help files with no progress.

**Comment by Wagnerp** [comment]
You may notice that I uploaded both the .cab and .mshc files. These were the original files posted on the original repo back in 2016.

**Comment by Smurf-IV** [comment]
`mshc` is a zipped format file.
Rename the one uploaded, expand and this just contains a `js` file. The `cab` is supposed to contain the source images etc. As both of these are only 5KB, then you can safely delete them as they are effectively empty !

I'll See what I can do with the `chm` and `chw` files, Assign this to me (And set labels etc) and I'll get on with it in my spare moments..

**Comment by Wagnerp** [comment]
Ahh, ok then. I'll make backup copies just in case. At the moment though, I'm just resolving some conflicts with the code.

**Comment by Wagnerp** [comment]
I may have decompiled the .chm file. Will upload directories shortly.

**Comment by Wagnerp** [comment]
@Smurf-IV 

I've somehow managed to 'crack' open the CHM file. I'll upload a new zip file to the Help directory containing its contents.

**Comment by Smurf-IV** [comment]
I've already done, but it would be useful to have it archived off independently..
Thanks for assigning.. Looks like I've got to perform some "real actions" now :-)

As for aesthetics... what should "we" put in the header and footer elements of each page ?
re:
![image](https://user-images.githubusercontent.com/2418812/35472938-4c538de8-0370-11e8-8cce-b1d1009c69c0.png)
and
![image](https://user-images.githubusercontent.com/2418812/35472940-5d2f331a-0370-11e8-9df8-2ad84aff9382.png)

i.e. another redirect for this Github site etc ?

**Comment by Wagnerp** [comment]
It maybe an upload of a new directory. Got 52,000+ files to push.

**Comment by Smurf-IV** [comment]
The number of files sounds about right (lot's of rubbish from the Dotfusctication and Sandcastle I summize)
![image](https://user-images.githubusercontent.com/2418812/35474326-a9c53c74-0384-11e8-95b5-a4e713ef4e5b.png)
The picture also includes the orginal files, so ignore the size !

I'l be putting in place another way of extracting the class diagrams etc, that the sandcastle build used to use.
But it will be kept simple and away from the main description mhtml (mht) files

**Comment by Wagnerp** [comment]
Yes, as the other way to extract it is to use 7zip. I  got the archive down from 850+ MB down to about 10 - 20 MB.

**Comment by Smurf-IV** [comment]
Ok, I have been fighting MHT, html, NDocs, and DocFX they all seem to come out at large file sizes (50MB++).
Currently DocFX is winning due ot the ability to create the API documentation AND also have MarkDown files for introdoctions.; But NDocs is in the process of being "Fixed" with Bug reports from me...

**Comment by Wagnerp** [comment]
Ok, the files I did manage to extract came out as nearly 30,000 HTML files, which were far too many to edit.

**Comment by Smurf-IV** [comment]
Current progress (What do you think about the logo ?)
![image](https://user-images.githubusercontent.com/2418812/35776180-c8494dfa-098f-11e8-9a59-2a0b303897b0.png)

**Comment by Wagnerp** [comment]
Looks good!

**Comment by Wagnerp** [comment]
Hi @Smurf-IV 

I was just browsing through the help files, have we changed the way [this](https://rawgit.com/Wagnerp/Krypton-NET-4.7/master/Help/Output/articles/palettes.html#built-in-palettes) works since its original incarnation? (Issue #113 & #77)

**Comment by Smurf-IV** [comment]
Not intentionally, But from a search I get
"RawGit is now in a sunset phase and will soon shut down. It's been a fun five years, but all things must end." https://rawgit.com/

Also, the URL is incorrect, as it should be `Krypton-NET-5.470` not `Krypton-NET-4.7`
https://rawgit.com/Wagnerp/Krypton-NET-4.7/master/Help/Output/articles/palettes.html#built-in-palettes
So the best thing is to relay the root via the project github - Can you do that please ?

**Comment by Wagnerp** [comment]
Hi, yes sure. Do you want me to re-write the create palettes page, as the information is incorrect. As for relaying the help, I'll look into it.

**Comment by Smurf-IV** [comment]
Yes please, for the rewrite.

**Comment by Wagnerp** [comment]
Will do, also where is the file for the main page, as I noticed a small grammatical mistake?

**Comment by Wagnerp** [comment]
Note to self: Add in VB .NET syntax
